lms/envs/aws.py

```diff
@@ -963,8 +963,7 @@
         'client_registration': {
             'client_id': FEATURES.get('OIDC_CLIENT_ID'),
             'client_secret': FEATURES.get('OIDC_CLIENT_SECRET'),
-            'redirect_uris': ['{}/openid/callback/login/'.format(LMS_ROOT_URL),
-                              '{}://{}/openid/callback/login/'.format(scheme, FEATURES.get('PREVIEW_LMS_BASE', ''))],
+            'redirect_uri': '%s://{}/openid/callback/login/' % scheme,
```

Comment:
Is it working correct?
probably you need `'{}://{}/openid/callback/login/'.format(scheme, LMS_ROOT_URL)`

Reply:
Nope, it's a feature of this setting, under edx-platform settings we set the scheme, then under django-oidc lib (which uses these settings) the next string formatting occurs here - 
https://github.com/raccoongang/django-oidc/blob/master/djangooidc/oidc.py#L47